tests/components/zha/common.py

```diff
@@ -173,6 +174,7 @@ async def async_enable_traffic(hass, zha_devices, enabled=True):
     """Allow traffic to flow through the gateway and the ZHA device."""
     for zha_device in zha_devices:
         zha_device.update_available(enabled)
```

Comment:
How long does this take on startup? Would be nice if we could stick to standard startup behaviours of blocking in async_init_entry()

Reply:
How long does what take on startup? The code tagged here is test stuff only. 

If you’re asking about device initialization it depends on the size of the network and if all mains powered devices are powered on and accessible. I need to check my logs but I think it’s between 45s and a couple minutes for larger networks.

Reply:
Somewhat misplaced comment indeed. Yes was more interested in how long after boot this signal normally is signalled. If its a more than a minute, then it would not be smart to wait in normal init.